tests/test_ballot_generator.py

```diff
@@ -422,185 +445,6 @@ def group_elements_by_mapping(element_list, mapping):
     )
 
 
-# def test_Cambridge_distribution():
```

Comment:
also looks like you overwrote some of the tweaks Zach and i made to make the cambridge test pass consistently, unless you two redid it after I left. looks like it was this in this commit: https://github.com/mggg/VoteKit/commit/798fef5c3296447c994ed04a20532c03f8a0469b